### ISSUE TITLE
Navigate through search results with arrow keys

### ISSUE THREAD

**Issue by AmadeusW** [CodeConnect/SourceBrowser#91]
When the focus is within the search box, pressing up and down keys should bring the focus down to the search results (or at least visually simulate focus). Pressing enter should navigate to the selected search result.

**Comment by JoshVarty** [comment]
It'd be nice if this was the case for the treeview as well. So you could expand and close nodes with the arrow keys too.

Maybe there's a way we can intelligently do this for both.

**Comment by AmadeusW** [comment]
Good idea. The search results are done using `a` and `span` elements, while the treeview operates on `li` tags. We could use a class attribute, or a common tag.

**Comment by JoshVarty** [comment]
We should first do #96 and then do this one, just to make sure nothing changes and causes us to do this twice.

**Comment by AmadeusW** [comment]
Since #96 will use icons for various file types, we could leverage the style in #71 and display icons next to search results - to help differentiate between results that are types (public, abstract, etc), methods, constructors, fields, constants etc.
